docs/additional-functionality/rapids-udfs.md

```diff
@@ -174,31 +173,51 @@ To enable _GPU Scheduling for Pandas UDF_, you need to configure your spark job
+3. Enable GPU Assignment(Scheduling) for Pandas UDF.
 
     ```shell
     ...
     --conf spark.rapids.python.gpu.enabled=true \
-    --conf spark.rapids.python.memory.gpu.pooling.enabled=false \
-    --conf spark.rapids.sql.exec.ArrowEvalPythonExec=true \
-    --conf spark.rapids.sql.exec.MapInPandasExec=true \
-    --conf spark.rapids.sql.exec.FlatMapGroupsInPandasExec=true \
-    --conf spark.rapids.sql.exec.AggregateInPandasExec=true \
-    --conf spark.rapids.sql.exec.FlatMapCoGroupsInPandasExec=true \
-    --conf spark.rapids.sql.exec.WindowInPandasExec=true
     ```
 
-Please note the data transfer acceleration only supports scalar UDF and Scalar iterator UDF currently. 
-You could choose the exec you need to enable.
+Please note the data transfer acceleration only supports `scalar UDF` and `Scalar iterator UDF` currently. 
+You could choose the exec see the section below.
 
 ### Other Configuration
 
 Following configuration settings are also for _GPU Scheduling for Pandas UDF_
-```
-spark.rapids.python.concurrentPythonWorkers
-spark.rapids.python.memory.gpu.allocFraction
-spark.rapids.python.memory.gpu.maxAllocFraction
-```
+
+1. Spark Plan support
+
+    ```
+    --conf spark.rapids.sql.exec.ArrowEvalPythonExec=true \
+    --conf spark.rapids.sql.exec.MapInPandasExec=false \
+    --conf spark.rapids.sql.exec.FlatMapGroupsInPandasExec=false \
+    --conf spark.rapids.sql.exec.AggregateInPandasExec=false \
+    --conf spark.rapids.sql.exec.FlatMapCoGroupsInPandasExec=false \
+    --conf spark.rapids.sql.exec.WindowInPandasExec=true
+    ```
+
+    These configs are the switches for each type of PandasUDF execution plan. Some of theme are set to false by default due to not supported or performance issue.
+
+2. Memory efficiency
+
+    ```
+    --conf spark.rapids.python.memory.gpu.pooling.enabled=false \
+    --conf spark.rapids.python.memory.gpu.allocFraction=0.1 \
+    --conf spark.rapids.python.memory.gpu.maxAllocFraction= 0.2 \
+    ```
+    Same to the [RMM pooling for JVM](../tuning-guide.md#pooled-memory), here the pooling serves the same way but for Python process.  `half of the rest GPU memory` will be used by default if it is not specified.
+
+3. Limit of concurrent Python processes
+
+    ```
+    --conf spark.rapids.python.concurrentPythonWorkers=2 \
+    ```
+    This parameter aims to limit the total concurrent running `Python process` in 1 Spark executor. This parameter is set to 0 by default which means there's not limit for concurrent Python workers. Note that for certain cases, setting this value too small may result a `hang` for your Spark job because a PandasUDF may produces multiple python process and each will try to acquire the python GPU process semaphore. This may bring a dead lock situation becasue a Spark job will not preceed until all its tasks are finished. For example, in the Pandas UDF stage, 2 tasks are running and each task launches 3 Python process and we set this parameter to 4.
```

Comment:
Wrap to 100 chars.
```suggestion
This parameter limits the total concurrent running `Python process` associated with a Spark executor.
The parameter defaults to 0 which means no limit. Note that for certain cases, setting this value too small
may result a hang for your Spark job.  A PandasUDF may produce multiple Python process and
each will try to acquire the Python GPU process semaphore. This may result in a deadlock situation
because a Spark job will not proceed until all the tasks are finished. For example, in the Pandas UDF stage
2 tasks are running, and each task launches 3 Python process and we set this parameter to 4.
```

Reply:
Updated.